README.md

```diff
@@ -78,20 +78,20 @@ cmake --build build --target package
 An example program called `sleigh-lift` has been included to demonstrate how to use the SLEIGH API. It takes a hexadecimal string of bytes and either disassembles it or lifts it to p-code. The program can be invoked like so, where the `action` argument must be either `disassemble` or `pcode`:
 
 ```sh
-sleigh-lift [action] [sla_file] [bytes] [address:OPTIONAL]
+sleigh-lift [action] [sla_file] [bytes] [-a address] [-f sla_path]
```

Comment:
`sla_path` is a bit ambiguous. Is it the path to the specific compiled `*.sla` file or is it to the root directory where we try to find `Ghidra/Processors/*/data/language`? The `-f` makes it seem like a "file" but then why is this here if we already specify `sla_file`? This should be either `sla_file_path` or `root_sla_dir_path` or something.

Reply:
perhaps `sla_dir` if it's mean to be a directory?

Reply:
Yeah it's the spot where we try to find `Ghidra/Processors/*/data/language`. I've renamed to `root_sla_dir` and changed the flag to `-p`.

9215bfb